src/gui/EffectView.cpp

```diff
@@ -116,7 +119,10 @@ EffectView::EffectView( Effect * _model, QWidget * _parent ) :
 			m_subWindow->hide();
 		}
 	}
-
+	
+	m_opacityEffect = new QGraphicsOpacityEffect(this);
```

Comment:
Does Qt `delete` this for us, or do we need one in the destructor?

Reply:
Qt should handle ``m_opacityEffect``'s deletion, because its parent is the ``EffectView``.